_docs_integrate/01-connector-tutorial.md

```diff
@@ -8,47 +8,173 @@ In this tutorial we go through the basic steps necessary to establish a Relation
 
 The following steps include small pieces of the Connector's API documentation that, when executed, fire requests on a Connector created for testing purposes. So if you don't have an own Connector installed, feel free to use the samples directly. Otherwise you can use your own Connector either with a REST client (e.g. [Insomnia](https://insomnia.rest/) or [Postman](https://www.postman.com/)) or by using the RapiDoc documentation (`/docs/rapidoc`) hosted on your Connector (you have to [enable docs on your connector]({% link _docs_integrate/11-connector-configuration.md %}#corehttpapi) for the documentation route to work).
 
+The payloads for the requests the will be performed during this tutorial contain placeholders marked with `<...>`. Please replace them with values.
+
 # Prerequisites
 
 -   If you want to use your own Connector for executing the examples
     -   [Install the Connector]({% link _docs_integrate/10-connector-installation.md %})
     -   Make sure the [Sync Module is disabled]({% link _docs_integrate/11-connector-configuration.md %}#sync) (because in this tutorial we will synchronize manually via the HTTP endpoint)
     -   Get the API key configured for the Connector. Ask the person who installed the Connector for it.
--   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device.
-
--   Diagramm
+-   You need the [Enmeshed App]({% link _pages/use.md %}) installed on your mobile device (App language set to `English`).
 
 # Establishing Relationships
 
 In order to communicate with another Identity, a Relationship to that Identity is required. In this first part of the tutorial you will learn how to establish a Relationship between your Connector and another Identity. In this case the other Identity will be the using the App, but it could be another Connector as well.
 
-## Connector: Create a Relationship Template
+## Connector: Create an Attribute
 
-Start by creating a so called Relationship Template on the Connector. You can do so by calling the `POST /api/v1/RelationshipTemplates/Own` route. Use the following JSON in the request body:
+As a prerequisite we provided an Attribute in the Connector that can be shared in the following step by executing `POST /api/v1/Attributes` with the following payload:
+
+```json
+{
+    "content": {
+        "@type": "IdentityAttribute",
+        "owner": "<your connector's identity address>",
+        "value": {
+            "@type": "DisplayName",
+            "value": "Connector Tutorial"
+        }
+    }
+}
+```
+
+You can query the Connectors Address under the route `/api/v1/Account/IdentityInfo`. If you are using our provided Connector please use `id16RPQxvUC8S5aTrRhs3yrDXq6cLkbbBsTY`.
```

Comment:
> If you are using our provided Connector

If you are using the Demo Connector of this Tutorial, the Address is `id16RPQxvUC8S5aTrRhs3yrDXq6cLkbbBsTY`.